codalab/apps/web/models.py

```diff
@@ -1601,9 +1605,26 @@ def dict_constructor(loader, node):
                 )
 
         participate_category = ContentCategory.objects.get(name="Participate")
-        Page.objects.create(category=participate_category, container=pc,  codename="get_data", competition=comp,
-                                   label="Get Data", rank=0, html=zf.read(comp_spec['html']['data']))
-        Page.objects.create(category=participate_category, container=pc,  codename="submit_results", label="Submit / View Results", rank=1, html="")
+        Page.objects.create(category=participate_category,
```

Comment:
I don't think github is correctly displaying this. PyCharm shows this as being broken up onto newlines already.